infra/build_specified_commit_test.py

```diff
@@ -59,17 +71,25 @@ def test_build_fuzzers_from_commit(self):
     old_commit = 'f79be4f2330f4b89ea2f42e1c44ca998c59a0c0f'
     new_commit = 'f50a39051ea8c7f10d6d8db9656658b49601caef'
     fuzzer = 'rules_fuzzer'
-    test_data = 'infra/yara_test_data'
+    test_data = os.path.join(
+        os.path.dirname(os.path.realpath(__file__)), 'testcases',
+        'yara_test_data')
     build_specified_commit.build_fuzzer_from_commit(
-        project_name, old_commit, 'tmp', sanitizer='address')
+        project_name, old_commit, TMP_DIR, sanitizer='address')
     old_error_code = helper.reproduce_impl(project_name, fuzzer, False, [], [],
                                            test_data)
     build_specified_commit.build_fuzzer_from_commit(
-        project_name, new_commit, 'tmp', sanitizer='address')
+        project_name, new_commit, TMP_DIR, sanitizer='address')
     new_error_code = helper.reproduce_impl(project_name, fuzzer, False, [], [],
                                            test_data)
     self.assertNotEqual(new_error_code, old_error_code)
 
 
 if __name__ == '__main__':
-  unittest.main()
+  if os.getcwd() != os.path.dirname(
+      os.path.dirname(os.path.realpath(__file__))):
+    print('Error: this script needs to be run from the OSS-Fuzz home directory')
+  else:
+    TMP_DIR = tempfile.mkdtemp()
```

Comment:
Did linting say anything about this? All caps is only for global constants.
Frankly I would expect this code to fail, I don't understand how TMP_DIR is defined on line 78.